tensorrt_llm/_torch/modules/attention.py

```diff
@@ -696,17 +712,7 @@ def _run_bmm():
             ])
             return fused_q
 
-        def _concat_kv_cache():
-            latent_cache = torch.concat([compressed_kv, k_pe], dim=-1)
-            return latent_cache
-
-        fused_q, latent_cache = maybe_execute_in_parallel(
-            _run_bmm,
-            _concat_kv_cache,
-            self.ln_events[0],
-            self.ln_events[1],
-            self.aux_stream,
-        )
+        fused_q = _run_bmm()
```

Comment:
Can you remove the `def _run_bmm()` function declaration? There is no need to put it in a function and call it here after removing the multi-stream.